SalesforceFoundation/k12-architecture-kit
# [K12] Fix Missing TSO Updates for Application, Test, and Test Scores

# Critical Changes

# Changes
**New List Views and Quick Actions**
In version 1.8, we made the following changes. If you installed an earlier version or installed on top of an existing org, consider making them to your org.

We introduced a K-12 All Applications list view on Application. If you choose to create a K-12 All Applications list view we recommend this configuration.

- Sharing Settings: All users can see this list view
- Filter by Owner: All attributes
- Display order: Application Name, Applicant, Applying To, Term, Application Type, Application Status, Application Date, Application Decision Date, Applicant Decision Date, Preparer
- See [Create or Clone a List View](https://powerofus.force.com/s/article/EDA-Display-Cust-Obj-Post-Rel).

We introduced a K-12 All Tests list view on Test. If you choose to create a K-12 All Tests list view, we recommend this configuration:

- Sharing Settings: All users can see this list view-
- Filter by Owner: All tests
- Display order: Test Name, Contact, Test Type, Test Date
- See [Create or Clone a List View](https://help.salesforce.com/articleView?id=customviews_lex.htm&type=5).
- Create a K12 Test Compact Layout with fields in this order: Test Name, Contact, Test Type, Test Date

We introduced a K-12 All Test Scores list view on Test Scores. If you choose to create a K-12 All Tests Scores list view, we recommend this configuration:

- Sharing Settings: All users can see this list view
- Filter by Owner: All test scores
- Display order: Test Score Name, Test Type, Test, Subject Area, Score, Percentile
- See [Create or Clone a List View](https://help.salesforce.com/articleView?id=customviews_lex.htm&type=5).
- Add K-12 All Test Scores as a related list on Test.
- For more information, see [Display a New Custom Object After a Product Release](https://powerofus.force.com/s/article/EDA-Display-Cust-Obj-Post-Rel).
- Create a K12 Test Compact Layout with fields in this order: Test Name, Contact, Test Type, Test Date

**Updates to the Test and Test Score Page Layouts**
We've added the Edit, New Note, New Case, Printable View, Change Owner, Clone, and Delete standard quick actions to the Test and Test Score page layouts. These quick actions are available from Salesforce Mobile and Lightning Experience Actions bar.
To use the new Case quick action:

- Ensure that your profile has proper permissions to edit quick actions.
- Edit the non-managed New Contact global action to specify a Record Type of Incident.
- On the Test and Test Score page layouts, replace the default New Case quick action with the 
- New Case quick action we've provided with this release. If you see two New Case quick actions, use the standard, non-managed quick action (NOT k12kit__NewContact quick action).

For more information, see[ Quick Action Considerations](https://help.salesforce.com/articleView?id=actions_considerations.htm&type=5).
# Issues Closed

# New Metadata
Page Layouts:

- K12 Kit Application Layout
- K12 Kit Test Layout
- K12 Kit Test Score Layout

List View:

- K12 All Application
- K12 All Tests
- K12 All Test Score

# Deleted Metadata

# Testing Notes
We cannot test trial org at this moment until this PR is merge to master - then test the trial org.
